modules/apps/object/object-rest-test/src/testFunctional/macros/CustomObjectAPI.macro

```diff
@@ -40,6 +187,25 @@ definition {
 		return ${staticObjectId};
 	}
 
+	macro updateObjectEntry {
```

Comment:
Thanks Kevin, it's using PUT here, what would you like me to update? The method name or the parameters inside the method, if the parameters, then I think it's better to keep it as it is now, since we can modify all the parameter we passed here.